src/DynamoCore/Logging/IAnalyticsClient.cs

```diff
@@ -15,6 +15,8 @@ public enum Categories
         Command,
         FileOperation,
         SearchUX,
+        Preferences,
```

Comment:
I think this is okay from a BC standpoint because the new entries are added to the end of the list?